MoneySplitter.Models/Models/FriendsOfUser.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Linq;
+
+namespace MoneySplitter.Models.Models
+{
+    public class FriendsOfUser
+    {
+        public int Id { get; set; }
+        public int UserId { get; set; }
+        public IQueryable<User> Friends { get; set; }
```

Comment:
I don't think we need such class. Just add IEnumerable<User> property to user model

Reply:
remove